app/controllers/concerns/checkout_callbacks.rb

```diff
@@ -2,6 +2,7 @@
 
 module CheckoutCallbacks
   extend ActiveSupport::Concern
+  include EnterprisesHelper
```

Comment:
You could just use this as a helper and then you don't need additional action and variable.
```suggestion
  helper EnterprisesHelper
```

Reply:
It seems impossible to call helper inside this module. 
I then have this error: `undefined method `helper' for CheckoutCallbacks:Module`

Any thoughts?

Reply:
Yes, I didn't see that the code was in a concern and not the controller. The `helper` method is only useful in controllers because they are connected to the view where we call the helper.

Reply:
Thanks @mkllnk ! 🙏